js/modules/k6/websocket/websocket_test.go

```diff
@@ -0,0 +1 @@
+package websocket
```

Comment:
There's a lot going on here.

But for starters, you gotta put that license header in there.